pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,196 @@
+//+build linux
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"net"
+	"os"
+
+	"github.com/ti-mo/conntrack"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/util/sysctl"
+)
+
+var _ ConnTrackDumper = new(connTrackDumper)
+
+type connTrackDumper struct {
+	nodeConfig  *config.NodeConfig
+	serviceCIDR *net.IPNet
+	connTrack   ConnTrackInterfacer
+}
+
+func NewConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, conntrack ConnTrackInterfacer) *connTrackDumper {
+	return &connTrackDumper{
+		nodeConfig,
+		serviceCIDR,
+		conntrack,
+	}
+}
+
+// DumpFlows opens netlink connection and dumps all the flows in Antrea ZoneID
+// of conntrack table, i.e., corresponding to Antrea OVS bridge.
+func (ctdump *connTrackDumper) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error) {
+	// Get netlink Connection to netfilter
+	err := ctdump.connTrack.Dial()
+	if err != nil {
+		klog.Errorf("Error when getting netlink conn: %v", err)
+		return nil, err
+	}
+
+	// Check value of setting net.netfilter.nf_conntrack_acct. Set to value 1, if it is not set.
```

Comment:
I don't really have a strong opinion here, so I'll defer to you, but have you considered setting `nf_conntrack_acct` once and not checking its value every time `DumpFlows` is called?